src/renderer/constants/routes.ts

```diff
@@ -5,6 +5,7 @@ export class Routes {
     public static VALIDATOR_DETAILS = "/details/:publicKey";
     public static BEACON_NODES = "/beacon-nodes";
     public static ADD_BEACON_NODE = "/add-beacon-node/:validatorKey";
+    public static ASSIGN_BEACON_NODE = "/assign-beacon-node/:validatorKey";
```

Comment:
what's the difference between ADD_BEACON_NODE and ASSIGN_BEACON_NODE route?

Reply:
add beacon add new beacon node to beacon node list (create docker or add URL), assign assing beacon node to a validator